datasets/wikiproject_taxonomy.halfak_20191202.yaml

```diff
@@ -43,7 +37,6 @@ Culture:
      - WikiProject Jewish Women
      - WikiProject Women in Music
      - WikiProject Women in Technology
-     - WikiProject Women's sport
```

Comment:
Same here.  Why remove this?

Reply:
Aha!  Duplicate?

Reply:
yeah -- just happened to notice the duplicate while looking at other things so removed it